pages/managing_relationships.md

```diff
@@ -125,7 +124,15 @@ And then the `Car` entity, as in the previous example:
     ? When you display this relationship with AngularJS, which field from 'Owner' do you want to use? id
 
 This will work as in the previous example, but you won't be able to add or remove cars from the `Owner` entity. On the generated AngularJS client UI you will have a dropdown in `Car` to select a `Owner`.
-This is not supported with JDL.
+This is the corresponding JDL:
```

Comment:
@MathieuAA I thought you always create bi directional in JHUML? has this been changed in versions after 1.6.1?